controllers/apps/apicast_controller_deployment_upgrade.mermaid.text

```diff
@@ -0,0 +1,25 @@
+%% Diagram can be generated using MERMAID
```

Comment:
Oh I have always wanted to try mermaid, looks good [mermaid.live](
https://mermaid.live/edit#pako:eNqFU8tuwjAQ_BXLUm8F1OeBQ6tAQoHyaKEcSsLBtbfgKnGixKFFgX_vAnEEKRU5WNbs7OzMKs4oDwXQOr24ILZk85gFhDNFPoDMQUHMNAiSJlLNSd8Z9a2O7SmkLrSOknqtFkAcMCkqX0l1LvUi_ajK0IBHxLzKw-Cg5yQzx6q-XIKnPv3wmy9YrD1F8BtrvLpWJDlLNHGWoPSMVCoPxMqsl84OFBD54SrACoEfmejkcbPvtZC3HgzXpJFpCKIzxHdnvCbNbBLhSgQIU2sYkYFrDwfO7ADdddhZMwZcGvHxQDd_XRklG3umV-7IeZ04EyPUNPKOOwIeKi59KDQSiJeSwyF1N7N1Jo-DzO4ZTssMfjrM1SpmtLPef4EIBhYrI9Q2QtPrUrZ2IdZxbfABl1TylBO7RuLZzZd5mve83eBNaUq3mNLL3kqRj4z2CqO3JYleIdE3RkNfnEg-I_uG_tbIXUmlswXvj0B6SfO_Gx9ctkU8qhcQgEfreBXwyVJfe9RTG6SyVIfjleK0ruMULmkaCdxF_kj34OYXhJk6Rg)

Reply:
it can be added directly in markdown now on github https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Reply:
I did not know about this, super nice!